### PR TITLE
Chore: Add redirect link to conversation in success message

### LINKED ISSUE #3063: chore: Redirect to new conversation on creating a new message from contacts view
when a conversation is created from the contacts tab, using the new message button. 

Show a link to the user in success response which they can click and get redirected to the newly created conversation

### PR DESCRIPTION
**Pull request by ximenasandoval**
# Pull Request Template

## Description

When creating a conversation from the contacts tab now we can go directly to the conversation by clicking the link in the success message.

Fixes #3063

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

https://user-images.githubusercontent.com/33427323/148610816-f1361469-d0af-4caf-94fa-b118582233f6.mov

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by pranavrajs** [comment]
Thanks, @ximenasandoval, It looks like a nice feature to have. Please see the comments below around the implementation.

File: app/javascript/dashboard/routes/dashboard/conversation/contact/ConversationForm.vue [hunk 1]
```diff
@@ -174,17 +174,25 @@ export default {
...
       this.$emit('success');
     },
-
+    getConversationAlert(conversationData) {
+      return `${this.$t('NEW_CONVERSATION.FORM.SUCCESS_MESSAGE')}
+           </br> <a href="/app/accounts/${
+             conversationData.account_id
+           }/conversations/${conversationData.id}">${this.$t(
+        'NEW_CONVERSATION.FORM.GO_TO_CONVERSATION'
+      )}</a>
```
Comment (pranavrajs) [inline app/javascript/dashboard/routes/dashboard/conversation/contact/ConversationForm.vue:183]: Passing an HTML seems a little off, can use additional params on the SnackBar component. Payload can be of the format.
```
{
   message: '',
   action: {
     type: 'link',
     to: '<link-to-url>'
   }
}
```

**Review by pranavrajs — CHANGES_REQUESTED** [review]
We can standardize the implementation so that we can solve similar use-cases easily.

**Comment by ximenasandoval** [comment]
Hi! Thanks for the review! I'll change it 🙌🏽

File: app/javascript/dashboard/i18n/locale/en/contact.json [hunk 2]
```diff
@@ -169,6 +169,7 @@
       "SUBMIT": "Send message",
       "CANCEL": "Cancel",
       "SUCCESS_MESSAGE": "Message sent!",
+      "GO_TO_CONVERSATION": "Go to conversation",
```
Comment (nithindavid) [inline app/javascript/dashboard/i18n/locale/en/contact.json:172]:
```suggestion
      "GO_TO_CONVERSATION": "VIEW",
```

File: app/javascript/dashboard/routes/dashboard/conversation/contact/ConversationForm.vue [hunk 3]
```diff
@@ -174,17 +174,25 @@ export default {
     onSuccess() {
       this.$emit('success');
     },
-
+    getConversationAlert(conversationData) {
+      return `${this.$t('NEW_CONVERSATION.FORM.SUCCESS_MESSAGE')}
+           </br> <a href="/app/accounts/${
```
Comment (nithindavid) [inline app/javascript/dashboard/routes/dashboard/conversation/contact/ConversationForm.vue]:
```suggestion
           <a href="/app/accounts/${
```

**Review by nithindavid — CHANGES_REQUESTED** [review]
@ximenasandoval  This is a nice change! I have added a suggestion to make the link and text come in same line. Please see if it makes sense

File: app/javascript/dashboard/i18n/locale/en/contact.json (hunk 2 above)
Comment (nithindavid) [reply app/javascript/dashboard/i18n/locale/en/contact.json:172]: Ignore if this is not in context with changes suggested by Pranav.

**Comment by ximenasandoval** [comment]
Hi! Thanks for the review 😄 sure, I'll change it to display the message and link together!

**Comment by ximenasandoval** [comment]
By double checking it would clash with the first changes suggested by Pranav 😲 it would change the payload.

**Comment by nithindavid** [comment]
> By double checking it would clash with the first changes suggested by Pranav 😲 it would change the payload.

You can Ignore it 👍

**Pull request merged**